### PR TITLE
xinfra monitor code formatting changes and Proper Indentations for MultiClusterTopicManagementService class

### PR DESCRIPTION
**Pull request by andrewchoi5**
No logic changes or functional changes. Only code formatting changes.

Signed-off-by: Andrew Choi <li_andchoi@microsoft.com>

### CODE REVIEW

File: src/main/java/com/linkedin/xinfra/monitor/services/MultiClusterTopicManagementService.java [hunk 1]
```diff
@@ -506,32 +505,28 @@ private void triggerPreferredLeaderElection(List<TopicPartitionInfo> partitionIn
...
+    private static void reassignPartitions(AdminClient adminClient, Collection<Node> brokers, String topicName,
+        int partitionCount) {
+
+      Map<TopicPartition, Optional<NewPartitionReassignment>> reassignments = new HashMap<>();
+      List<Integer> targetReplicas = new ArrayList<>();
       for (Node broker : brokers) {
-        brokersMetadata.$plus$eq(new BrokerMetadata(broker.id(), Option$.MODULE$.apply(broker.rack())));
-      }
-      scala.collection.Map<Object, Seq<Object>> assignedReplicas =
-          AdminUtils.assignReplicasToBrokers(brokersMetadata, partitionCount, replicationFactor, 0, 0);
```
Comment (gitlw) [inline src/main/java/com/linkedin/xinfra/monitor/services/MultiClusterTopicManagementService.java:515]: Previously the call of AdminUtils.assignReplicasToBrokers would try to distribute the replicas among the set of brokers, more or less evenly.

But now every partition will have a replica on all the brokers?

Comment (andrewchoi5) [reply src/main/java/com/linkedin/xinfra/monitor/services/MultiClusterTopicManagementService.java:515]: Let's try AdminUtils for the replica assignments to brokers.

**Comment by andrewchoi5** [comment]
@gitlw https://github.com/linkedin/kafka-monitor/pull/317
addressed them here ^

File: src/main/java/com/linkedin/xinfra/monitor/services/ConsumeService.java [hunk 2]
```diff
@@ -40,7 +40,7 @@
 import org.apache.kafka.common.metrics.Metrics;
 import org.apache.kafka.common.metrics.MetricsReporter;
 import org.apache.kafka.common.metrics.Sensor;
-import org.apache.kafka.common.metrics.stats.Total;
+import org.apache.kafka.common.metrics.stats.CumulativeSum;
```
Comment (smccauliff) [inline src/main/java/com/linkedin/xinfra/monitor/services/ConsumeService.java:43]: Do you know if the changes to the metrics up stream will change the computation of any of the metrics produced by the monitor?

Comment (andrewchoi5) [reply src/main/java/com/linkedin/xinfra/monitor/services/ConsumeService.java]: Addressed in the above PR

Comment (smccauliff) [reply src/main/java/com/linkedin/xinfra/monitor/services/ConsumeService.java]: I don't have a specific change I'm asking for.  I'm asking if you understand the changes that were done to Kafka that resulted in Total becoming CumulativeSum.  Are there any other side effects as part of that change to open source Kafka?

Comment (andrewchoi5) [reply src/main/java/com/linkedin/xinfra/monitor/services/ConsumeService.java]: See KIP-488 --
https://cwiki.apache.org/confluence/display/KAFKA/KIP-488%3A+Clean+up+Sum%2CCount%2CTotal+Metrics 

In previous cases, no consistency existed in naming convention with a missing metric, the quadrant for which was redundantly covered. With the new changes in Kafka, the metrics including the CumulativeSum for "Sum" are vividly named with all quadrants covered in a unique way -- no ambiguity +structure of titles show a clear pattern that maps to correct metrics.

`Non-sampled Sum Metric`:
- (deprecated) org.apache.kafka.common.metrics.stats.Total
- (new) org.apache.kafka.common.metrics.stats.CumulativeSum

Does that answer your question?

image

**Pull request merged**